charts/traefik/values.insecure.yaml.gotmpl

```diff
@@ -32,16 +33,18 @@ extraObjects:
   kind: Ingress
   metadata:
     name: traefik-dashboard
+    namespace: traefik
     annotations:
-      traefik.ingress.kubernetes.io/router.entrypoints: web,websecure  # allow http(s) for local deployment
-      traefik.ingress.kubernetes.io/router.middlewares: default-traefik-basic-auth@kubernetescrd  # namespace + middleware name
+      traefik.ingress.kubernetes.io/router.entrypoints: websecure  # allow http(s) for local deployment
```

Comment:
If we disable `http`, then we can delete this file (`valiues.insecure.yaml.hotmpl`), I think

Reply:
makes sense,

Reply:
will address this